packages/webassemblyjs/test/fixtures/type-errors/add-mixed-operators/exec.tjs

```diff
@@ -0,0 +1,6 @@
+it("should throw an error during instantiation when type checking fails", () => {
+  assert.throws(
```

Comment:
Maybe we can take advange of the `throws.txt` we have for tokensizer. It has less noise that a entire test.

Reply:
Ah yep I didnt consider this. Would have to modifiy the index.js accordingly

Reply:
Well it's not easy actually, this packages uses the `exec.js` to instantiate the module. Let's leave it as is for now.